### DISCUSSION TITLE
How to deploy an internationalized app to Firebase?

### DISCUSSION THREAD

**Discussion by nullvoidundefined** [i18next/next-i18next#2000] [General]
I have a Next.js app that I need to deploy to Firebase. Deployment is seamless using Firebase's experimental Next framework support, but once I add `next-i18next` to the pipeline, Firebase returns 500 errors. Has anyone had any luck deploying an `next-i18next` site to Firebase?

**Comment by adrai** [comment]
Sorry, never used Firebase...
But I suspect Firebase does not support internationalized routing: https://nextjs.org/docs/advanced-features/i18n-routing
https://stackoverflow.com/questions/73084816/nextjs-with-next-i18next-and-firebase-functions-routes-all-pages-to-404-when-loc

**Reply by Hello-Misha** [reply]
I have the same problem and have spent more than a week to solve it and fail.I have the same problem and have spent more than a week to solve it, but unfortunately had failed.
Can you please suggest any way to deploy (except Vercel)?

**Reply by leach-mrt** [reply]
me tooooo

**Reply by takuma7** [reply]
I was searching for the same issue. It looks like this needs to be resolved so we can override ignore settings: https://github.com/firebase/firebase-tools/issues/5607
